concordia/admin/__init__.py

```diff
@@ -310,13 +310,13 @@ class ItemAdmin(admin.ModelAdmin):
 
     actions = (publish_item_action, unpublish_item_action)
 
-    def get_deleted_objects(self, objs, request):
+    def get_deleted_objects(self, objs, request):  
         deleted_objects = [str(obj) for obj in objs]
-        model_count = {'Items': len(deleted_objects)}
-        perms_needed = []
+        perms_needed = set()
```

Comment:
This doesn't check that the user has permissions to delete related objects. I think in this case we could either hard-code the list or do something like this:

```python
from django.contrib.auth import get_permission_codename

…

perms_needed = set()
for model in (Item, Asset, Transcription):
    p = '%s.%s' % (model._meta.app_label, get_permission_codename('delete', model._meta))
    if not user.has_perm(p):
        perms_needed.add(model._meta.verbose_name)
```

Reply:
done